analysis/src/CompletionBackEnd.ml

```diff
@@ -1566,7 +1568,7 @@ let printConstructorArgs argsLen ~asSnippet =
   else ""
 
 let completeTypedValue ~env ~envWhereCompletionStarted ~full ~prefix
-    ~expandOption =
+    ~expandOption ~includeLocalValues ~completionContext =
```

Comment:
I'm letting this function swell a bit for now. Current plan is to refactor when I've landed most cases, including expression completion.